diagnostic.md

```diff
@@ -32,15 +36,15 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+val() and text()
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+DOM is a model that connect html files and javascript file. It models the HTML files to node.
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.  A web browser implements this interface to render web pages for you to view and interact with.